dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/persister/EnrollmentPersister.java

```diff
@@ -163,16 +152,15 @@ protected TrackerType getType() {
 
   @Override
   protected void persistOwnership(TrackerPreheat preheat, Enrollment entity) {
-    if (isNew(preheat, entity.getUid())) {
```

Comment:
Why is the `isNew` deleted? Was this wrong?

Reply:
It was not wrong but the logic to make it work with relationships was really complicated.
I put it back but changed the implementation to use the strategy in `TrackerBundle` instead of checking the `TrackerPreheat`